.kokoro

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
If this is for Google (since your description says Kokoro), why not Blaze and TAP? It's better supported and more functional, imho.